geode-core/src/main/java/org/apache/geode/internal/tcp/Connection.java

```diff
@@ -1698,6 +1698,19 @@ private void readMessages() {
           }
           return;
         } catch (IOException e) {
+          // "Socket closed" check needed for Solaris jdk 1.4.2_08
```

Comment:
Wait, where did this sneak in from. JDK 1.4 isn't even supported anymore.

Reply:
it's on the `develop` branch. I had removed it in a previous commit on this branch but then reverted my changes.

Reply:
there I removed this spurious (repeated) chunk of code. This method now matches what's on the `develop` branch